File: Java/src/entities/Vehicle.java
package entities;

public class Vehicle {
    private int id;
    private int type;
    private float fuel;
    private int site_id;

    public Vehicle() {}

    public Vehicle(int id) {
        this.id = id;
    }

    public Vehicle(int id, int type, float fuel, int site_id) {
        this.id = id;
        this.type = type;
        this.fuel = fuel;
        this.site_id = site_id;
    }

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public int getType() {
        return type;
    }

    public void setType(int type) {
        this.type = type;
    }

    public float getFuel() {
        return fuel;
    }

    public void setFuel(float fuel) {
        this.fuel = fuel;
    }

    public int getSite_id() {
        return site_id;
    }

    public void setSite_id(int site_id) {
        this.site_id = site_id;
    }
}

File: realApi/src/Entity/Vehicle.php
<?php
namespace App\Entity;
use Doctrine\ORM\Mapping as ORM;
use Symfony\Component\Validator\Constraints as Assert;
/**
 * @ORM\Entity
 * @ORM\Table(name="vehicle")
 */
class Vehicle {
    /**
     * @ORM\Column(type="integer")
     * @ORM\Id
     * @ORM\GeneratedValue(strategy="AUTO")
     */
    private $id;

    /**
     * @ORM\Column(type="text", length=100, nullable=false)
     * @Assert\NotBlank()
     *
     */
    private $registrationNumber;

    /**
     * @ORM\Column(type="text", nullable=true)
     * @Assert\NotBlank()
     */
    private $fuel;

    /**
     * @ORM\ManyToOne (targetEntity="CategoryVehicle")
     * @ORM\JoinColumn(name="category_vehicle_id", referencedColumnName="id")
     */
    private $categoryVehicle;

    /**
     * @ORM\ManyToOne (targetEntity="Team", inversedBy="vehicles")
     * @ORM\JoinColumn(name="team_id", referencedColumnName="id")
     */
    private $team;

    /**
     * @return mixed
     */
    public function getId()
    {
        return $this->id;
    }

    /**
     * @return mixed
     */
    public function getRegistrationNumber()
    {
        return $this->registrationNumber;
    }

    /**
     * @param mixed $registrationNumber
     * @return Vehicle
     */
    public function setRegistrationNumber($registrationNumber)
    {
        $this->registrationNumber = $registrationNumber;
        return $this;
    }

    /**
     * @return mixed
     */
    public function getFuel()
    {
        return $this->fuel;
    }

    /**
     * @param mixed $fuel
     * @return Vehicle
     */
    public function setFuel($fuel)
    {
        $this->fuel = $fuel;
        return $this;
    }

    /**
     * @return mixed
     */
    public function getCategoryVehicle()
    {
        return $this->categoryVehicle;
    }

    /**
     * @param mixed $categoryVehicle
     * @return Vehicle
     */
    public function setCategoryVehicle($categoryVehicle)
    {
        $this->categoryVehicle = $categoryVehicle;
        return $this;
    }

    /**
     * @return mixed
     */
    public function getTeam()
    {
        return $this->team;
    }

    /**
     * @param mixed $team
     * @return Vehicle
     */
    public function setTeam($team)
    {
        $this->team = $team;
        return $this;
    }
}
